.codespellignore

```diff
@@ -0,0 +1,4 @@
+fo
+raison
+opne
```

Comment:
I'm not seeing any direct occurence of this. The closest I'm finding is:

> .//zydis/Zydis.c:35676:    ZYDIS_MAKE_SHORTSTRING("loopne"),

Should the issue be fixed instead?

Reply:
This is from actual code
```
nativeunwind/elfunwindinfo/elfehframe.go
113:    opNE         expressionOpcode = 0x2e
308:            case op == opDeref, op >= opRot && op <= opNE:
```